src/configs/routesConfigs/elevate-entity-management.json

```diff
@@ -8,7 +8,7 @@
             "orchestrated": false,
             "targetPackages": [
                 {
-                    "basePackageName": "entity_management",
+                    "basePackageName": "entity-management",
```

Comment:
@adithyadinesh0412 not required seperate file

Reply:
its needed @priyanka-TL